packages/web/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
why do you need the license there as well ?

Reply:
Yes, I removed it